bin/build-debug.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ $npm_package_browserify_options ]; then
+  options=$npm_package_browserify_options
+else
+  options=$NPM_PACKAGE_BROWSERIFY_OPTIONS
```

Comment:
Can you explain why you need to check both here? I know a lot of stuff on Windows is case-insensitive...is NPM setting these vars in uppercase? Do you think we can  be assured that only one of them is set?

Might be cleaner to do it like this:

```
# Account for both Unix (lowercase) and Windows (uppercase).
OPTIONS=${npm_package_browserify_options:−$NPM_PACKAGE_BROWSERIFY_OPTIONS}
```